CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing
+
+This project is licensed under the [Apache 2.0 license](LICENSE) and accept
+contributions via GitHub pull requests. This document outlines some of the
+conventions on development workflow, commit message formatting, contact points
+and other resources to make it easier to get your contribution accepted.
+
+To maintain a safe and welcoming community, all participants must adhere to the
+project's [Code of Conduct](code-of-conduct.md).
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
+
+## Email and Chat
+
+The project currently uses the [Kubernetes Slack](https://slack.k8s.io/):
+- [#prometheus-operator](https://kubernetes.slack.com/archives/CFFDS2Z7F)
+- [#prometheus-operator-dev](https://kubernetes.slack.com/archives/C01B03QCSMN)
+
```

Comment:
@paulfantom worth mentioning https://github.com/prometheus-operator/kube-prometheus/discussions here?

Reply:
Yes, good catch!

Reply:
I have rewrote this section and modeled it after prometheus site - https://prometheus.io/community/